ecs-cli/modules/clients/aws/ecs/client.go

```diff
@@ -391,6 +394,39 @@ func (c *ecsClient) GetEC2InstanceIDs(containerInstanceArns []*string) (map[stri
 	return containerToEC2InstanceMap, nil
 }
 
+// DescribeContainer Instances returns a Map with key container instance ARN and values list of attributes
+func (c *ecsClient) GetAttributesFromDescribeContainerInstances(containerInstanceArns []*string) (map[string][]*string, error) {
+	descrContainerInstancesoutputMap := map[string][]*string{}
+	for i := 0; i < len(containerInstanceArns); i += ecsChunkSize {
+		var chunk []*string
+		if i+ecsChunkSize > len(containerInstanceArns) {
+			chunk = containerInstanceArns[i:len(containerInstanceArns)]
+		} else {
+			chunk = containerInstanceArns[i : i+ecsChunkSize]
+		}
+
+		descrContainerInstances, err := c.client.DescribeContainerInstances(&ecs.DescribeContainerInstancesInput{
+			Cluster:            aws.String(c.config.Cluster),
+			ContainerInstances: chunk,
+		})
+		if err != nil {
+			log.WithFields(log.Fields{
```

Comment:
Instead of logging the error, I'd rather the error be returned, and the command quit. If the customer inputs invalid container instances or something else goes wrong, the command should fail- all of our commands follow that style.